src/Compilers/VisualBasic/Portable/BoundTree/BoundSequencePointExpression.vb

```diff
@@ -9,12 +9,6 @@ Imports Microsoft.CodeAnalysis.VisualBasic.Syntax
 Namespace Microsoft.CodeAnalysis.VisualBasic
 
     Partial Friend Class BoundSequencePointExpression
-
-#If DEBUG Then
-        Private Sub Validate()
```

Comment:
We should probably add `HasValidate="true"` to `BoundSequencePointExpression` in VB's `BoundNodes.xml` instead

Reply:
Nice! I didn't know this was a thing!